google-cloud-firestore/lib/google/cloud/firestore/query.rb

```diff
@@ -361,9 +361,11 @@ def limit num
         # If the current query already has specified `start_at` or
         # `start_after`, this will overwrite it.
         #
-        # The values provided here are for the field paths provides to `order`.
+        # The values provided here are for the field paths provided to `order`.
         # Values provided to `start_at` without an associated field path
-        # provided to `order` will result in an error.
+        # provided to `order` will result in an error. If a {DocumentSnapshot}
+        # object is provided the `document_id` value will be used and should be
```

Comment:
We should not need to expose the fact that we use the `document_id`. If a DocumentSnapshot is provided, we will do this automatically without enforcing this on the user.

Reply:
Okay.

Reply:
Removed.